src/x86/uarch.c

```diff
@@ -2,6 +2,12 @@
 
 #include <cpuinfo.h>
 #include <x86/api.h>
+#include <x86/cpuid.h>
+
+CPUINFO_INTERNAL bool cpuinfo_x86_detect_hybrid_ecore(){
+	if (((cpuid(0x1A).eax >> 24) & 0xFFF) == 0x20) return true;
+	return false;
+}
```

Comment:
This should be `cpuidex(0x1A, 0)`; per [page 40 of this doc](https://cdrdv2-public.intel.com/790021/architecture-instruction-set-extensions-programming-reference.pdf#page=40), it requires `ecx = 0`:

> Native Model ID Enumeration Leaf (Initial EAX Value = 1AH, ECX = 0)